src/hashtable.py

```diff
@@ -0,0 +1,26 @@
+class HashTable(object):
+
+    def __init__(self, buckets=1024):
+        self.buckets = buckets
+        self.init_table()
+
```

Comment:
If the only time you use `init_table()` is in `__init__`, and it's automatically called when an instance is created, why separate them?